docs/src/pages/[platform]/components/searchfield/examples/SearchFieldThemeExample.tsx

```diff
@@ -0,0 +1,33 @@
+import {
+  SearchField,
+  ThemeProvider,
+  Theme,
+  useTheme,
+} from '@aws-amplify/ui-react';
+
+export const SearchFieldThemeExample = () => {
+  const { tokens } = useTheme();
+
+  const theme: Theme = {
+    name: 'searchfield-theme',
+    tokens: {
+      components: {
+        searchfield: {
+          color: { value: 'red' },
+          search: {
```

Comment:
I think this should be `button: {` right? Looking at the searchField.ts file below

Reply:
thanks that has been updated to `button`